### PR TITLE
DR-3036: Card column breakpoints

### PR DESCRIPTION
**Pull request by 7emansell**
## Ticket:

Resolves JIRA ticket [DR-3036](https://newyorkpubliclibrary.atlassian.net/browse/DR-3036)

## This PR does the following:

- Adds new column breakpoints to the home page swimlanes

### Checklist:

- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.
- [x] I have updated the CHANGELOG.md.

### CODE REVIEW

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Only change is to try to figure out the flicker situation for the desktop view.

File: app/components/swimlanes/swimLanes.tsx [hunk 1]
```diff
@@ -19,7 +19,12 @@ import { DC_URL } from "../../config/constants";
 import useBreakpoints from "../../hooks/useBreakpoints";
 
 const SwimLanes = ({ lanesWithNumItems }) => {
-  const { isLargerThanLargeTablet } = useBreakpoints();
+  const { isLargerThanLargeTablet, isLargerThanLargeMobile } = useBreakpoints();
+  const getNumColumns = () => {
+    if (isLargerThanLargeTablet) return 4;
+    if (isLargerThanLargeMobile) return 2;
+    return 1;
```
[the remark this answers is not in the document]
Comment (7emansell) [reply app/components/swimlanes/swimLanes.tsx]: Passing it down as a prop is the only thing that resolves the flicker (not setting it as state in a `useEffect`, or reversing the logic) but it does kind of seem like overkill

File: app/components/swimlanes/swimLanes.tsx [hunk 2]
```diff
@@ -16,9 +16,9 @@ import {
 import styles from "./Swimlanes.module.css";
 import { imageURL } from "../../utils/utils";
 import { DC_URL } from "../../config/constants";
-import useBreakpoints from "../../hooks/useBreakpoints";
+import useBreakpoints from "app/hooks/useBreakpoints";
 
-const SwimLanes = ({ lanesWithNumItems }) => {
+const SwimLanes = ({ numColumns, lanesWithNumItems }) => {
   const { isLargerThanLargeTablet } = useBreakpoints();
```
Comment (EdwinGuzman) [inline app/components/swimlanes/swimLanes.tsx:22]: You don't need to import `useBreakpoints` here anymore.

**Review by EdwinGuzman — APPROVED** [review]
Minor, remove the unnecessary import in `swimLanes.tsx`.

This works better but I get that it's not ideal and the cost is low. This is something I think Marty and I should get working properly in the DS and allow components to be more flexible with responsive styles. And also to try to move away from the `useNYPLBreakpoint`/`useBreakpoint` hook because we've noticed it causes a noticeable flicker.

**Comment by 7emansell** [comment]
Would it be better to do it as media queries entirely? @EdwinGuzman

File: app/components/swimlanes/swimLanes.tsx (hunk 2 above)
Comment (7emansell) [reply app/components/swimlanes/swimLanes.tsx:22]: It's used in another place

Comment (EdwinGuzman) [reply app/components/swimlanes/swimLanes.tsx:22]: Ah, the tooltip.

**Comment by EdwinGuzman** [comment]
Yes, I think it's a good case where CSS can and should be able to handle the media queries and responsive styles. It _could_ work with a CSS module but not sure if you want to introduce that now. I'd prefer if devs can add the chakra responsive style object but `SimpleGrid` does not currently allow that.

I'm okay with this approach if you are.

**Review by avertrees — APPROVED** [review]

**Pull request merged**